content/faq/_index.md

```diff
@@ -140,3 +140,12 @@ Release:  %{?autorel}%{!?autorel:1}
 This construct uses `autorel` macro if it's defined, and if it's not, it sets release to 1.
 
 For `%changelog`, you don't need to include the changelog file upstream and you can have it downstream only, which makes sense - changelog is specific to a release.
+
+### How do I install dependencies for my commands in packit-service?
+
+We are running all commands, defined by you, in a sandbox which is locked-down.
+At the moment we don't have any mechanism for you to define the dependencies
+you need and us making them available for you.
+
+In the mean time we are solving these requests one by one, so [please reach out
+to us](#how-can-i-contact-you).
```

Comment:
```suggestion
In the mean time we are solving these requests one by one, so [please reach out
to us](https://github.com/packit/sandcastle/issues/new).
```

Reply:
I'm sorry but I'm going to deny this: people can also send an email, IRC message or start an issue - it's really up to them to pick their favourite communication channel, you are forcing them here to open an issue on a repo which they don't understand probably at all. Feel free to open a PR for the 'contacting' section to expand it with links to repos and a brief description.

Reply:
Ok, I was looking at it from my POV - I'd rather be given a textbox - where I'd just write "I need foobar" and hit 'Submit' button - than need to write to some IRC channel or send email to some list - but that's just my social phobia. :-)